features/events/traps/blueprint-trapd-listener.xml

```diff
@@ -43,6 +56,23 @@
             <cm:property name="password" value="admin"/>
         </cm:default-properties>
     </cm:property-placeholder>
+    
+  
+
+    <!-- A canonical source for this Minion's identity -->
+    <bean id="minionIdentity" class="org.opennms.minion.core.impl.MinionIdentityImpl">
+       <argument value="[[location]]"/>
+       <argument value="[[id]]"/>
+    </bean>
+    <service ref="minionIdentity" interface="org.opennms.minion.core.api.MinionIdentity"/>
+
+    <!-- REST Client used to communicate with the OpenNMS controller -->
+    <bean id="restClient" class="org.opennms.minion.core.impl.ScvEnabledRestClientImpl">
```

Comment:
Is there any specific reason why we need to redefine this bean instead of pulling it from the service registry?